sdk/core/Azure.Core/tests/common/ModelReaderWriter/ServiceModels/ComputeSku.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.Core.Tests.Models.ResourceManager.Compute
+{
+    /// <summary> Describes a virtual machine scale set sku. NOTE: If the new VM SKU is not supported on the hardware the scale set is currently on, you need to deallocate the VMs in the scale set before you modify the SKU name. </summary>
+    public partial class ComputeSku
+    {
+        /// <summary> Initializes a new instance of ComputeSku. </summary>
+        public ComputeSku()
+        {
+        }
+
+        /// <summary> Initializes a new instance of ComputeSku. </summary>
+        /// <param name="name"> The sku name. </param>
+        /// <param name="tier"> Specifies the tier of virtual machines in a scale set.&lt;br /&gt;&lt;br /&gt; Possible Values:&lt;br /&gt;&lt;br /&gt; **Standard**&lt;br /&gt;&lt;br /&gt; **Basic**. </param>
```

Comment:
I'm guessing this is generated and is a swagger issue?

Reply:
yes